include/odp/api/spec/crypto.h

```diff
@@ -262,6 +262,24 @@ typedef enum {
 	 */
 	ODP_AUTH_ALG_ZUC_EIA3,
 
+	/** MD5 algorithm */
+	ODP_AUTH_ALG_MD5,
```

Comment:
MD5 has been deprecated for some years now. Is this being added for historical reasons?

Reply:
It has been though there is still hardware supporting it. For the sake of legacy systems.